charts/prefect-server/templates/_validation.tpl

```diff
@@ -1,22 +1,33 @@
 {{- define "prefect-server.validatePrefectVersion" -}}
-{{- if .Values.backgroundServices.runAsSeparateDeployment -}}
```

Comment:
this `if` statement logic is handled where the validation is actually called -- now under the `background-services-deployment.yaml`